documentation/src/main/doc/modules/ROOT/pages/getting-started.adoc

```diff
@@ -55,8 +55,8 @@ Finally, you need an _empty_ `beans.xml`. Copy the following content to `src/mai
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="
       http://xmlns.jcp.org/xml/ns/javaee
-      http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd">
-
+      http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd"
+  bean-discovery-mode="annotated">
```

Comment:
It is amusing, the previous line says "_empty_ `beans.xml`" :-)

I agree with recommending annotated, for sure.